neurobooth_os/tasks/utils.py

```diff
@@ -288,17 +288,3 @@ def advance():
     key = get_keys(keyList=["space"])
     if key == ["space"]:
         return True
-
-
-def run_task(task, prompt=True):
```

Comment:
Thanks for fixing this as well, I was going to remove this piece of dead code in some future utils fix!